scIB/metrics.py

```diff
@@ -361,35 +360,44 @@ def get_isolated_labels(adata, label_key, batch_key, cluster_key, n, verbose):
         print(f"no isolated labels with less than {n} batches")
     return labels
 
-def score_isolated_label(adata, label_key, batch_key, cluster_key,
-                         label, cluster=True, verbose=False, **kwargs):
+
+def score_isolated_label(adata, label_key, cluster_key, label, cluster=True, verbose=False, **kwargs):
     """
     compute label score for a single label
     params:
         cluster: if True, use clustering approach, otherwise use silhouette score approach
     """
-    
-    import sklearn.metrics as scm
     adata_tmp = adata.copy()
-    
-    # cluster optimizing over cluster with largest number of isolated label per batch
+
     def max_label_per_batch(adata, label_key, cluster_key, label, argmax=False):
+        """cluster optimizing over cluster with largest number of isolated label per batch"""
         sub = adata.obs[adata.obs[label_key] == label].copy()
         label_counts = sub[cluster_key].value_counts()
+        print(label_counts)
```

Comment:
this looks like it shouldn't be turned on by default but only if verbose or sth, no?

Reply:
Yep, removed those